org.gnuhealth.MyGNUHealth.yaml

```
@@ -0,0 +1,366 @@
+id: org.gnuhealth.MyGNUHealth
```

```suggestion
id: org.gnuhealth.mygnuhealth
```

And the file renamed. It's the one used upstream.

---

that was not a suggestion. you should use the same app-id that is used upstream. I case I was unclear.

---

Done